Setup.hs

```diff
@@ -69,6 +70,20 @@ copyHook' pd lbi hooks flags = do
     -- , extraDocFiles = []
     }
 
+-- We only want to write interfaces if installing the executable.
+-- If we're installing *just* the library, the interface files are not needed
```

Comment:
I am not so sure about this sentence.
Even from the Agda library, the `Agda_datadir` is available (via `Paths_Agda`), and the primitive modules are loaded even if one calls the Agda driver from the library.
Now if the Agda library was installed in superuser mode to a superuser location, the ordinary user cannot use the Agda library because loading the primitive modules will fail.  The reason for the latter is that Agda wants to put the generated interface files for the primitive modules into the `Agda_datadir` which resides in superuser land.

The best fix would probably call the _Agda library_ from Setup and not the _Agda executable_ to generate the interface files.
However, I do not even want to research whether this is possible, and also not stretch the uses of Setup further.

So let's ignore the problem.

The PR is in any case an improvement over the status quo.

If anything to wish for:  Maybe it would make sense that Setup prints a warning "Skipping the generation of the Agda library interface files" rather than doing this just silently.

Reply:
> Setup prints a warning "Skipping the generation of the Agda library interface files" rather than doing this just silently.

I added this.